packages/tools/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -2,6 +2,7 @@ import { existsSync, mkdirSync } from "fs";
 import { join, relative } from "path";
 import { getBabelInputPlugin, getBabelOutputPlugin } from "@rollup/plugin-babel";
 import commonjs from "@rollup/plugin-commonjs";
+import image from "@rollup/plugin-image";
```

Comment:
Why is this here?

Reply:
All the other imports are alphabetically imported, so i thought it was good to fix this file.

Reply:
Ah, I missed that it moved. Perfect